test/freenet/support/MultiValueTableTest.java

```diff
@@ -199,11 +198,11 @@ public void testGetAll() {
 		methodMVTable = fillMultiValueTable(sampleObjects);
 		Iterator<Object> iter;
 		Enumeration<Object> methodEnumeration;
-		for(int i=0;i<sampleObjects.length;i++) {
-			iter = ((List<Object>)(sampleObjects[i][1])).iterator();
-			methodEnumeration = methodMVTable.getAll(sampleObjects[i][0]);
-			while(iter.hasNext())
-				assertEquals(methodEnumeration.nextElement(),iter.next());
+		for (Object[] sampleObject : sampleObjects) {
```

Comment:
Same here; cast `sampleObjects`, change the type of `sampleObject`, remove cast in line below.